credentials/apps/badges/admin.py

```diff
@@ -388,14 +388,24 @@ def template_link(self, instance):
         """
         Interactive link to parent (badge template).
         """
-        url = reverse("admin:badges_credlybadgetemplate_change", args=[instance.template.pk])
+        if instance.template.origin == CredlyBadgeTemplate.ORIGIN:
```

Comment:
Perhaps we should create a dictionary where the key is ORIGIN and the value is reverse_name? 
This would make adding new providers easier without having to add a new elif. It would also be nice to add some kind of default reverse_name to avoid errors.